learning_resources.md

```diff
@@ -101,6 +101,10 @@
   Introduction to JavaScript and web development.
 * [You Don't Know JavaScript by Kyle Simpson](https://github.com/getify/You-Dont-Know-JS)
   In-depth free eBook on JavaScript.
+  
+### Node
```

Comment:
This is cutting the JS section in half. Could you move the new Node section after the ES6 subsection?